docs/mappingservice-crd-reference.md

```diff
@@ -0,0 +1,93 @@
+# MappingService Custom Resource Reference
+
+## Simple CR Example
+
+```yaml
+apiVersion: saas.3scale.net/v1alpha1
+kind: MappingService
+metadata:
+  name: simple-example
+spec:
+  image:
+    tag: mapping-service-v3.8.0-r3
+  replicas: 2
+  env:
+    apiHost: http://system:3000
+    previewBaseDomain: stg-saas.3scale.net
```

Comment:
We are creating all examples with fake domain `example.3scale.net`.

Can you please update all referenced real possible domains (simple, full, and cr example)?

Reply:
6a8e13e